src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,118 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BeforeAfter;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise1 extends BeforeAfter {
+    private WebDriverWait webDriverWait;
+    private WebElement webElement;
+    static final SoftAssertions SOF = new SoftAssertions();
+
+    @Test
+    public void exercise1() {
```

Comment:
We should name the test classes and test methods according to what we're checking in them, please, try to think of better namings